terraform/protoc-gen-terraform-teleport.yaml

```diff
@@ -126,16 +134,14 @@ exclude_fields:
     - "RoleV6.Spec.Deny.Namespaces"
 
     # SessionRecordingConfig
-    - "SessionRecordingConfigV2.Metadata.Name" # It's a signleton resource
+    - "SessionRecordingConfigV2.Metadata.Name" # It's a singleton resource
 
     # User
     - "UserSpecV2.LocalAuth" # We do not need to set user auth data
     - "UserSpecV2.Expires"   # Those fields are Teleport side meta, we do not need to change them via Terraform
     - "UserSpecV2.CreatedBy"
     - "UserSpecV2.Status"
 
-name_overrides:
-
```

Comment:
I know we are not using it but can we keep it here? Just to create awareness that this feature exists
It's useful for handling field renames

Reply:
Oh sure, I actually did not intend to delete this!